### DISCUSSION TITLE
How set max-height for b-form-select's options wrapper?

### DISCUSSION THREAD

**Discussion by AtousaDarabi** [bootstrap-vue/bootstrap-vue#6553] [Q&A] [answered]
I used b-form-select.
When the options are so many, height if the options' wrapper gets higher till touch the screen and after that scrolls.
I want to set that height be maximum 200px.
How can I  make it?

**Comment by Hiws** [comment, marked as answer]
You can't control the size of the "dropdown" when opening a select.
It's completely controlled and rendered by the browser.
`<b-select>` is a simple wrapper around a `<select>`, and native selects are notoriously bad customization wise. 

You can for example see the difference in rendering if you open it on a desktop and a mobile browser.

There's a few different "hacks" by utilizing the `size` attribute, but nothing as you describe it to my knowledge.
